lib/app/features/chat/messages/views/components/mesage_timestamp/mesage_timestamp.dart

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+class MessageTimeStamp extends StatelessWidget {
```

Comment:
This widget contains not only the timestamp but also `iconMessageRead`, which may be confusing. Please split it into two different widgets or rename this one.